include/range/v3/view/any_view.hpp

```diff
@@ -192,10 +259,17 @@ namespace ranges
                 }
                 virtual Ref read() const override { return *current(); }
                 virtual void next() override { ++current(); }
+                std::size_t size() const // override-ish
+                {
+                    return static_cast<std::size_t>(ranges::size(range()));
+                }
             };
 
-            template<typename Ref, category Cat = category::forward>
-            struct any_cursor_interface
+            template<typename Ref, category Cat = category::forward, typename enable = void>
+            struct any_cursor_interface;
+
+            template<typename Ref, category Cat>
+            struct any_cursor_interface<Ref, Cat, meta::_t<std::enable_if<(Cat & category::mask) == category::forward>>>
```

Comment:
`any_cursor_interface<Ref, Cat, meta::if_c<(Cat & category::mask) == category::forward>>`. And elsewhere below.

Reply:
done